miniapp/miniapp_triangular_multiplication.cpp

```diff
@@ -169,9 +189,6 @@ struct triangularMultiplicationMiniapp {
                   << pika::get_os_thread_count() << " " << backend << std::endl;
       }
 
-      b.copyTargetToSource();
-
-      // (optional) run test
       if ((opts.do_check == dlaf::miniapp::CheckIterFreq::Last && run_index == (opts.nruns - 1)) ||
           opts.do_check == dlaf::miniapp::CheckIterFreq::All) {
         DLAF_UNIMPLEMENTED("Check");
```

Comment:
Confused by the title of the PR: "Miniapp trmm check". Is this supposed to enable results checking for the miniapp or is that to be done in a separate PR? If separate PR, could you please update the title and description of the PR to say what the goal of this PR is?